src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -81,21 +81,38 @@ internal static void ValidateIS(string value, string name)
         }
 
         internal static void ValidateLO(string value, string name)
+        {
+            ValidationError error = ValidateLO(value);
```

Comment:
Why don't we throw directly in the helper `ValidateLO` method?

Reply:
ValidateLO is used on 2 places:
1. STOW: when customer store dicom instance, it validate  DicomElement of VR LO.
2. AddCustomTag: when custom add custom tag, it validate privateCreator (whose VR is LO) 

Those logic is same except error message, I didn't figure out a simpler way to do this.